core/kernel/tee_ta_manager.c

```diff
@@ -21,15 +21,11 @@
 #include <mm/core_mmu.h>
 #include <mm/mobj.h>
 #include <mm/vm.h>
-#include <pta_stats.h>
```

Comment:
_pta_stats.h_ is needed for `struct pta_stats_ta`

Reply:
Right ... So it should be 
```
#ifdef CFG_TA_STATS
#include <pta_stats.h>
#endif
````
no ?

Reply:
Sometimes it might be warranted, but usually it would just make the code harder to read and maintain. We probably need to accept that Coverity sometimes is counterproductive.

Reply:
I don't like adding #ifdef for header files inclusion.
Coverity can be configured to not warn on such, no?

Reply:
I can extend the Coverity code coverage with additional compilation parameters and artifacts. If I add a compilation run with `CFG_TA_STATS=y`, I will get rid of it.